docs/user-guide/api.md

```diff
@@ -1,17 +1,36 @@
-# API Design
+# API
 
-Our API was designed with three principles in mind:
+Screwdriver APIs and the data models around them are documented via [Swagger]. This prevents out-of-date documentation, enables clients to be auto-generated, and most importantly exposes a human-readable interface. All of our Swagger can be found [here](https://api.screwdriver.cd/v4/swagger.json).
```

Comment:
Do we really want to send them to the swagger JSON, instead of the swagger UI? I think I'd link the JSON elsewhere, like a "create your own client" section.